src/client/search/TermItem.js

```diff
@@ -1,13 +1,45 @@
 import React, { Component } from 'react';
 
 const TermItem = ({ onDeleteClick, onToggleParamTypeClick, query, source, paramTypes, id }) => (
-  <div data-id={id} className="ui multiple compact dropdown labeled icon violet button"
+  <div data-id={id} className="ui multiple dropdown labeled icon violet button"
     onClick={(e) => e.stopPropagation()}
   >
-    <span>
-      <i className={`${source} icon`}></i>
-      {query}
-    </span>
+    <div className="ui grid">
+      <div className="two wide middle aligned column" style={{ paddingLeft: 0 }}>
+        <i className={`large blue ${source} icon`}></i>
+      </div>
+      <div className="thirteen wide center aligned column">
```

Comment:
doiesn't like fourteen... found that within components the grid size is 15 not 16 :P